gestalt/provider/README.md

```diff
@@ -0,0 +1,56 @@
+# Provider
```

Comment:
Just a note to add some info about it using proper jsonPath for traversing the secret.